io/FV3GFS_io.F90

```diff
@@ -640,6 +640,14 @@ subroutine sfc_prop_restart_read (Sfcprop, Atm_block, Model, fv_domain)
         Sfcprop(nb)%oro_uf(ix)    = oro_var2(i,j,16)
         Sfcprop(nb)%landfrac(ix)  = oro_var2(i,j,17) !land frac [0:1]
         Sfcprop(nb)%lakefrac(ix)  = oro_var2(i,j,18) !lake frac [0:1]
+
+        Sfcprop(nb)%lakedepth(ix) = oro_var2(i,j,19) !lake depth [m]    !YWu
+
+        if(Model%lkm.ne.1) then   !to avoid program crash when flake not called
+           Sfcprop(nb)%lakefrac(ix)  = 0.0
```

Comment:
Why lakefrac has to be set to zero when lkm.ne.1? It is a good idea to have the model running with nonzero lakefrac and without lake model.